pkg/configuration/configuration.go

```diff
@@ -114,6 +121,12 @@ func initConfig(secret map[string]string) *Config {
 	c.host.SetTypeByDefaultValue(true)
 	c.setConfigDefaults()
 
+	if c.host.GetString(varForbiddenUsernamePrefixes) != "" {
+		c.forbiddenUsernamePrefixes = strings.FieldsFunc(c.host.GetString(varForbiddenUsernamePrefixes), func(c rune) bool {
+			return c == ','
+		})
+	}
```

Comment:
could you please add a test for this part?

Reply:
Added test to explicitly check this.